hack/create_self_signed_cert.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -ex
+
+TMPDIR=${TMPDIR:-"/tmp/k8s-webhook-server/serving-certs"}
+SERVICE=${SERVICE:-"openstackcontrolplane"}
```

Comment:
Should this just be 'openstack' (for the openstack-operator). As we will have other CRD here as well.

Or do we instead need a certificate per CRD within the operator?